packages/rocketchat-webrtc/WebRTCClass.coffee

```diff
@@ -803,9 +803,11 @@ WebRTC = new class
 
 
 Meteor.startup ->
-	RocketChat.Notifications.onUser 'webrtc', (type, data) =>
-		if not data.room? then return
+	Tracker.autorun ->
```

Comment:
Sorry, I'm not sure I understand what you're saying. The autorun needs to keep running if a user isn't logged in yet (ie. Meteor.userId() is false), but will be logged in soon, say in the case of Sandstorm where login takes ~50ms. Stopping it would defeat the purpose, I think?

The concept for this was copied from https://github.com/RocketChat/Rocket.Chat/blob/a5cb22bb0017f4c39654bf1e2895ae64acb0339b/client/notifications/notification.coffee where you're already doing this logic.